packages/flutter/test/widgets/physical_model_test.dart

```diff
@@ -55,16 +55,23 @@ void main() {
   testWidgets('PhysicalModel - creates a physical model layer when it needs compositing', (WidgetTester tester) async {
     debugDisableShadows = false;
     await tester.pumpWidget(
-      MediaQuery(
-        data: const MediaQueryData(devicePixelRatio: 1.0),
-        child: Directionality(
-          textDirection: TextDirection.ltr,
-          child: PhysicalModel(
-            shape: BoxShape.rectangle,
-            color: Colors.grey,
-            shadowColor: Colors.red,
-            elevation: 1.0,
-            child: Material(child: TextField(controller: TextEditingController())),
+      Localizations(
```

Comment:
Would it be simpler to just wrap all of the subtree that's actually being tested with a MaterialApp()? All the stubbery is impressive,  but maybe it's not worth the trouble.

Reply:
I concur!